/*
 * This file was automatically generated by EvoSuite
 * Sat Apr 22 02:53:50 KST 2017
 */

package com.sleepycat.util;

import static org.evosuite.runtime.EvoAssertions.assertThrownBy;
import static org.junit.Assert.assertNull;
import static org.junit.Assert.assertSame;
import static org.junit.Assert.fail;

import org.evosuite.runtime.EvoRunner;
import org.evosuite.runtime.EvoRunnerParameters;
import org.junit.Test;
import org.junit.runner.RunWith;

@RunWith(EvoRunner.class) @EvoRunnerParameters(useVNET = true) 
public class ExceptionUnwrapper_ESTest10 extends ExceptionUnwrapper_ESTest_scaffolding10 {




  @Test
  public void test3()  throws Throwable  {
      Exception exception0 = new Exception();
      Exception exception1 = ExceptionUnwrapper.unwrap(exception0);
      assertSame(exception0, exception1);
  }

}
